docs/src/main/asciidoc/security-oidc-code-flow-authentication-concept.adoc

```diff
@@ -242,7 +633,9 @@ quarkus.http.auth.permission.public.policy=permit
 You may also need to set `quarkus.oidc.authentication.cookie-path` to a path value common to all the application resources which is `/` in this example.
 For more information, see the <<oidc-cookies, Cookies>> section.
 
-Note that some OpenID Connect providers do not support https://openid.net/specs/openid-connect-session-1_0.html#RPLogout[RP-Initiated Logout] specification (possibly because it is still technically a draft) and do not return an OpenID Connect well-known `end_session_endpoint` metadata property. However, it should not be a problem since these providers' specific logout mechanisms may only differ in how the logout URL query parameters are named.
+[NOTE]
+--
+Some OpenID Connect providers do not support https://openid.net/specs/openid-connect-session-1_0.html#RPLogout[RP-Initiated Logout] specification (possibly because it is still technically a draft) and do not return an OpenID Connect well-known `end_session_endpoint` metadata property. However, it is not be a problem since these providers' specific logout mechanisms only differ in how the logout URL query parameters are named.
```

Comment:
Remove assumptions (probably...) that might not be true in the near future.

```suggestion
Some OIDC providers do not support link:https://openid.net/specs/openid-connect-session-1_0.html#RPLogout[RP-initiated logout] specification and do not return an OpenID Connect well-known `end_session_endpoint` metadata property. 
However, this is not a problem for Quarkus because the specific logout mechanisms of such OIDC providers only differ in how the logout URL query parameters are named.
```